main.tf

```diff
@@ -236,6 +238,7 @@ resource "azurerm_kubernetes_cluster_node_pool" "this" {
   enable_auto_scaling   = true
   max_count             = each.value.max_count
   min_count             = each.value.min_count
+  node_labels           = each.value.labels
```

Comment:
i think you need to add this to https://github.com/Azure/terraform-azurerm-avm-ptn-aks-production/blob/6cd415a9fcac51b146fcdad20db2dbfd67784440/locals.tf#L36 as well

Reply:
You are probably correct. But that list is incomplete.
Does it mean we have a bug ?

Compare the 2 lists:

https://github.com/Azure/terraform-azurerm-avm-ptn-aks-production/blob/6cd415a9fcac51b146fcdad20db2dbfd67784440/variables.tf#L102-L115

and

https://github.com/Azure/terraform-azurerm-avm-ptn-aks-production/blob/6cd415a9fcac51b146fcdad20db2dbfd67784440/locals.tf#L32-L46

Reply:
I added the node labels in the locals. We will address the flattening of the other missing parameters in a different PR

Reply:
added it an issue here https://github.com/Azure/terraform-azurerm-avm-ptn-aks-production/issues/109